app/code/Magento/MediaGalleryApi/Model/Directory/Command/CreateByPathInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryApi\Model\Directory\Command;
+
+/**
+ * Create folder by provided path
```

Comment:
This doesn't sound like a necessary API for MediaGallery, more like a low-level API of file system.
Why is it needed?
I'd expect that MediaGallery allows to create/add an asset, and the folder should be created automatically if it's needed (using file system or whatever storage API). MediaGallery should not care about folders, only about assets, IMO.

Reply:
Same for all other `Model/Directory` interfaces. I don't think they are needed.

Reply:
These services have been provided to support Media Gallery functionality (both existing and new) allowing creating/deleting directories in the media storage without uploading/deleting assets